runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/wire/LengthPrefixUnknownCoders.java

```diff
@@ -16,43 +16,36 @@
 
 package org.apache.beam.runners.fnexecution.wire;
 
-import com.google.common.collect.Sets;
-import java.util.Set;
+import java.util.function.Predicate;
 import org.apache.beam.model.pipeline.v1.RunnerApi;
 import org.apache.beam.model.pipeline.v1.RunnerApi.Coder;
 import org.apache.beam.model.pipeline.v1.RunnerApi.Components;
-import org.apache.beam.model.pipeline.v1.RunnerApi.MessageWithComponents;
 import org.apache.beam.runners.core.construction.ModelCoders;
 import org.apache.beam.sdk.coders.ByteArrayCoder;
 import org.apache.beam.sdk.coders.LengthPrefixCoder;
 
-/**
- * Utilities for replacing or wrapping unknown coders with {@link LengthPrefixCoder}.
- */
+/** Utilities for replacing or wrapping unknown coders with {@link LengthPrefixCoder}. */
 public class LengthPrefixUnknownCoders {
   /**
    * Recursively traverse the coder tree and wrap the first unknown coder in every branch with a
```

Comment:
Is there a way to make this comment clearer that the components builder is modified?